### DISCUSSION TITLE
MIB Upgrade: instruction on how to move backups

### DISCUSSION THREAD

**Discussion by andreabafile** [Mr-MIBonk/M.I.B._More-Incredible-Bash#159] [Ideas]
It would be great if you could clarify within the Upgrade instructions [here](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Update-&-Uninstall-M.I.B) how to move correctly the backups that were done with previous versions. Is it sufficient to copy over the backup folder?

Thank you.

**Comment by harman-f** [comment]
yes, moving them from old to new version (SD card) would be enough.
The logic how the folders are created stay the same, so folders will be compatable.

I will try to add a few lines about how to do this.
